### PR TITLE
image-magick was added

### PR DESCRIPTION
**Pull request by ahmetak4n**
- [imagemagick](https://github.com/imagemagick/imagemagick): new package
- version: 7.1.1-18
- #6063 

- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

**Comment by ahmetak4n** [comment]
The lib version include '-' like '7.1.1-18'. That's why Wolfictl Lint throw error? How can I pass this step?

**Review by amouat — CHANGES_REQUESTED** [review]

File: image-magick.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: image-magick
+  version: 18
```
[the remark this answers is not in the document]
Comment (amouat) [reply image-magick.yaml:3]: Ah - I see the problem, the dash was causing issues. Let me check on the solution.

**Review by amouat — CHANGES_REQUESTED** [review]

File: image-magick.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amouat) [reply image-magick.yaml:3]: I think we can use something like this to replace the dash:

```
var-transforms:
  - from: ${{package.version}}
    match: \.
    replace: "-"
    to: mangled-package-version
```

From https://github.com/wolfi-dev/os/blob/a500b7eb8e85b52f589b5febb994ae7126eeba81/re2.yaml#L9-L13

Note that the fetch URL needs to be changed to reference this variable.

File: image-magick.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  epoch: 0
+  description: "library for displaying, converting, and editing raster image and vector image files"
+  copyright:
+    - license: ImageMagick Studio LLC
```
Comment (amouat) [inline image-magick.yaml:7]: I think we should use the SPDX identifier, which is just "ImageMagick" in this case.

**Comment by amouat** [comment]
There's a few issues with dependencies in this PR, so I've actually started a new one. I think the configure script is clever and just drops features if the library isn't present, which isn't really what we want.

File: image-magick.yaml (hunk 2 above)
Comment (ahmetak4n) [reply image-magick.yaml:7]: They mention that`the license is compatible with the GPL V3` in https://imagemagick.org/script/license.php we can use GPLV3

File: image-magick.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahmetak4n) [reply image-magick.yaml]: Thank for the guidence. I was added var-transform block 🙏

File: image-magick.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    strip-prefix: 7.1.1-
+    tag-filter: 7.1.1-
```
[the remark this answers is not in the document]
Comment (ahmetak4n) [reply image-magick.yaml]: prefix and tag-filter were removed.

File: image-magick.yaml (hunk 2 above)
Comment (ahmetak4n) [reply image-magick.yaml]: I changed the license as GPLV3

**Pull request closed without merging**